source/_integrations/command_line.markdown

```diff
@@ -136,3 +136,13 @@ binary_sensor:
     payload_on: 'active'
     payload_off: 'inactive'
 ```
+
+## Services
+
+Available services: `reload`.
+
+#### Service `command_line.reload`
```

Comment:
Wrong heading level.

The `Services` header is level 2, while this header is beneath it and has level 4.

@boc-the-git Please correct that in a new PR. Thanks 👍